### PR TITLE
contracts-bedrock: add base sepolia deploy config

### PR DESCRIPTION
**Pull request by tynes**
**Description**

Sources the deploy config from https://github.com/base-org/contract-deployments/blob/main/sepolia/2023-09-26-deploy/deploy-config/deploy-config.json
We need to double check that this is up to date, ie it is possible that
some config has changed over time. The problem variables that we care
about from within this config are the constructor args to the
L2OutputOracle:

- cfg.l2OutputOracleSubmissionInterval
- cfg.l2BlockTime
- cfg.l2OutputOracleProposer
- cfg.l2OutputOracleChallenger
- cfg.finalizationPeriodSeconds

If any of these values changed, they should be caught by the state diff
tests written in `superchain-ops` for performing the upgrade.

This deploy config is required to deploy a new L2OutputOracle
implementation for Base Sepolia as part of the Extended Pause Upgrade.

### CODE REVIEW

File: packages/contracts-bedrock/deploy-config/base-sepolia.json
Comment (maurelian) [inline packages/contracts-bedrock/deploy-config/base-sepolia.json:1]: Setting up my review to validate the necessary config values:

```
➜ cd superchain-registry/superchain/configs/sepolia
➜ cat base.yaml| yq .system_config_addr
0xf272670eb55e895584501d564AfEB048bEd26194


➜ cast call 0xf272670eb55e895584501d564AfEB048bEd26194 'l2OutputOracle()' --rpc-url https://ethereum-sepolia.publicnode.com
0x00000000000000000000000084457ca9d0163fbc4bbfe4dfbb20ba46e48df254

```

File: packages/contracts-bedrock/deploy-config/base-sepolia.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+  "maxSequencerDrift": 600,
+  "sequencerWindowSize": 3600,
+  "channelTimeout": 300,
+
+  "p2pSequencerAddress": "0xb830b99c95Ea32300039624Cb567d324D4b1D83C",
+  "batchInboxAddress": "0xFf00000000000000000000000000000000084532",
+  "batchSenderAddress": "0x6CDEbe940BC0F26850285cacA097C11c33103E47",
+
+  "l2OutputOracleSubmissionInterval": 120,
```
Comment (maurelian) [inline packages/contracts-bedrock/deploy-config/base-sepolia.json:19]:
```
➜ cast call 0x84457ca9d0163fbc4bbfe4dfbb20ba46e48df254 'submissionInterval()(uint)' --rpc-url https://ethereum-sepolia.publicnode.com
120
```

File: packages/contracts-bedrock/deploy-config/base-sepolia.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  "l2OutputOracleStartingBlockNumber": 0,
+  "l2OutputOracleStartingTimestamp": -1,
+
+  "l2OutputOracleProposer": "0x20044a0d104E9e788A0C984A2B7eAe615afD046b",
+  "l2OutputOracleChallenger": "0xDa3037Ff70Ac92CD867c683BD807e5A484857405",
+
+  "finalizationPeriodSeconds": 12,
```
Comment (maurelian) [inline packages/contracts-bedrock/deploy-config/base-sepolia.json:26]:
```
cast call 0x84457ca9d0163fbc4bbfe4dfbb20ba46e48df254 'finalizationPeriodSeconds()(uint)' --rpc-url https://ethereum-sepolia.publicnode.com
12
```

File: packages/contracts-bedrock/deploy-config/base-sepolia.json [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  "p2pSequencerAddress": "0xb830b99c95Ea32300039624Cb567d324D4b1D83C",
+  "batchInboxAddress": "0xFf00000000000000000000000000000000084532",
+  "batchSenderAddress": "0x6CDEbe940BC0F26850285cacA097C11c33103E47",
+
+  "l2OutputOracleSubmissionInterval": 120,
+  "l2OutputOracleStartingBlockNumber": 0,
+  "l2OutputOracleStartingTimestamp": -1,
+
+  "l2OutputOracleProposer": "0x20044a0d104E9e788A0C984A2B7eAe615afD046b",
+  "l2OutputOracleChallenger": "0xDa3037Ff70Ac92CD867c683BD807e5A484857405",
```
Comment (maurelian) [inline packages/contracts-bedrock/deploy-config/base-sepolia.json:24]:
```
➜ cast call 0x84457ca9d0163fbc4bbfe4dfbb20ba46e48df254 'challenger()(address)' --rpc-url https://ethereum-sepolia.publicnode.com
0xDa3037Ff70Ac92CD867c683BD807e5A484857405
```

File: packages/contracts-bedrock/deploy-config/base-sepolia.json [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+  "p2pSequencerAddress": "0xb830b99c95Ea32300039624Cb567d324D4b1D83C",
+  "batchInboxAddress": "0xFf00000000000000000000000000000000084532",
+  "batchSenderAddress": "0x6CDEbe940BC0F26850285cacA097C11c33103E47",
+
+  "l2OutputOracleSubmissionInterval": 120,
+  "l2OutputOracleStartingBlockNumber": 0,
+  "l2OutputOracleStartingTimestamp": -1,
+
+  "l2OutputOracleProposer": "0x20044a0d104E9e788A0C984A2B7eAe615afD046b",
```
Comment (maurelian) [inline packages/contracts-bedrock/deploy-config/base-sepolia.json:23]:
```
cast call 0x84457ca9d0163fbc4bbfe4dfbb20ba46e48df254 'proposer()(address)' --rpc-url https://ethereum-sepolia.publicnode.com
0x20044a0d104E9e788A0C984A2B7eAe615afD046b

```

File: packages/contracts-bedrock/deploy-config/base-sepolia.json [hunk 5]
```diff
@@ -0,0 +1,62 @@
+{
+  "finalSystemOwner": "0x608081689Fe46936fB2fBDF7552CbB1D80ad4822",
+  "portalGuardian": "0xA9FF930151130fd19DA1F03E5077AFB7C78F8503",
+
+  "l1StartingBlockTag": "safe",
+
+  "l1ChainID": 11155111,
+  "l2ChainID": 84532,
+  "l1BlockTime": 12,
+  "l2BlockTime": 2,
```
Comment (maurelian) [inline packages/contracts-bedrock/deploy-config/base-sepolia.json:9]:
```
➜ cast call 0x84457ca9d0163fbc4bbfe4dfbb20ba46e48df254 'l2BlockTime()(uint)' --rpc-url https://ethereum-sepolia.publicnode.com
2

```

**Review by maurelian — APPROVED** [review]

File: packages/contracts-bedrock/deploy-config/base-sepolia.json [hunk 6]
```diff
@@ -0,0 +1,62 @@
+{
+  "finalSystemOwner": "0x608081689Fe46936fB2fBDF7552CbB1D80ad4822",
+  "portalGuardian": "0xA9FF930151130fd19DA1F03E5077AFB7C78F8503",
```
Comment (maurelian) [inline packages/contracts-bedrock/deploy-config/base-sepolia.json]: I think this is causing the failure:

```suggestion
```

Comment (maurelian) [reply packages/contracts-bedrock/deploy-config/base-sepolia.json]: Committing this to see.

Comment (tynes) [reply packages/contracts-bedrock/deploy-config/base-sepolia.json]: I am unclear what the diff is here

Comment (tynes) [reply packages/contracts-bedrock/deploy-config/base-sepolia.json]: Ah ok github bug, the fix is already merged

Comment (maurelian) [reply packages/contracts-bedrock/deploy-config/base-sepolia.json]: deletes that line. The var name is now `superchainConfigGuardian`

Comment (maurelian) [reply packages/contracts-bedrock/deploy-config/base-sepolia.json]: and it's listed below.

**Pull request merged**